app/scripts/controllers/transactions.js

```diff
@@ -577,6 +594,13 @@ angular.module('openhimConsoleApp')
 
       // execute refresh if no errors
       if($scope.ngError.hasErrors === false) {
+        
+        // Set dateApplied flag if a date filter is being applied
+        if($scope.settings.filter.startDate || $scope.settings.filter.endDate) {
+          $scope.dateApplied = true;
+        } else {
+          $scope.dateApplied = false;
+        }
```

Comment:
It doesn't look like this code has any value within this function. This check can be made directly in the filtersApplied function (ln: 695)